mockgcp/mock_http_roundtrip.go

```diff
@@ -63,6 +63,8 @@ import (
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockgkehub"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockgkemulticloud"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mockkms"
+	_ "github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mocklogging_config"
+	_ "github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/mocklogging_metrics"
```

Comment:
with the import in register.go, you no longer need to update the mock_http_rountrip.go.